lib/components/element.rb

```diff
@@ -50,6 +56,14 @@ def self.element(name, multiple: false, &config)
     # rubocop:enable Metrics/MethodLength
     # rubocop:enable Metrics/PerceivedComplexity
 
+    class << self
```

Comment:
Should we move the other class methods here as well? I don't particularly care either way.

Reply:
I think nesting all class methods within a `class << self; end` definition makes the code less readable. To define the private class method similar to the other class methods I now used the `private_class_method :xxx` definition.